### ISSUE TITLE
adjust height

### ISSUE THREAD

**Issue by eskaigualker** [Vchouliaras/jquery.animateSlider.js#4]
I'm trying to adjust the height of each slide (showing only one at a time). Is there a way to implement a callback after each animation is ended?

thank you

**Comment by The1JJF1** [comment]
No, callback is impossible, because my code have many bugs. I stopped working on this project half year ago because exist many better plugins than my.
